apps/backend/src/events/events.service.ts

```diff
@@ -11,12 +12,28 @@ export class EventsService {
   constructor(readonly prisma: PrismaService) {}
 
   async create(data: CreateEventDto, userId: string): Promise<Event> {
-    return await this.prisma.event.create({
-      data: {
-        ...data,
-        ownerId: userId,
-      },
-    });
+    const { drinkActions, ...eventData } = data;
+    try {
+      return await this.prisma.event.create({
+        data: {
+          ...eventData,
+          ownerId: userId,
+          drinkActions: {
+            create:
+              drinkActions?.map((action) => ({
+                ...action,
+                drink: { connect: { id: action.drinkId } },
+              })) || [],
+          },
+        },
+        include: { drinkActions: true },
+      });
```

Comment:
If you try this, you'll see that prisma will throw an error that there's an unexpected field (`drinkId`) on the drinkAction object. I'm not sure why this is a problem for prisma, but it works if you remove that field from the action parameter in line 23-26 (you can use [object destructuring](https://byby.dev/ts-object-destructuring))